Mage.Sets/src/mage/cards/y/YarusRoarOfTheOldGods.java

```diff
@@ -0,0 +1,118 @@
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.CENTAUR);
+        this.subtype.add(SubType.DRUID);
+        this.power = new MageInt(4);
+        this.toughness = new MageInt(4);
+
+        // Other creatures you control have haste.
+        this.addAbility(new SimpleStaticAbility(new GainAbilityControlledEffect(
+                HasteAbility.getInstance(), Duration.WhileOnBattlefield,
+                StaticFilters.FILTER_PERMANENT_CREATURES, true)));
+        // Whenever one or more face-down creatures you control deal combat damage to a player, draw a card.
+        this.addAbility(new DealCombatDamageControlledTriggeredAbility(
+                new DrawCardSourceControllerEffect(1), filter)
+                .setTriggerPhrase("Whenever one or more face-down creatures you control deal combat damage to a player, "));
+        // Whenever a face-down creature you control dies, return it to the battlefield face down under its owner's control if it's a permanent card, then turn it face up.
+        this.addAbility(new DiesCreatureTriggeredAbility(
+                new YarusRoarOfTheOldGodsEffect(), false, filter, true));
+    }
+
+    private YarusRoarOfTheOldGods(final YarusRoarOfTheOldGods card) {
+        super(card);
+    }
+
+    @Override
+    public YarusRoarOfTheOldGods copy() {
+        return new YarusRoarOfTheOldGods(this);
+    }
+}
+
+class YarusRoarOfTheOldGodsEffect extends OneShotEffect {
+
+    YarusRoarOfTheOldGodsEffect() {
+        super(Outcome.PutCardInPlay);
+        this.staticText = "return it to the battlefield face down under its owner's control " +
+                "if it's a permanent card, then turn it face up";
+    }
+
+    private YarusRoarOfTheOldGodsEffect(final YarusRoarOfTheOldGodsEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public YarusRoarOfTheOldGodsEffect copy() {
+        return new YarusRoarOfTheOldGodsEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        Card card = game.getCard(getTargetPointer().getFirst(game, source));
+        if (controller == null || card == null || !card.isPermanent(game)) {
+            return false;
+        }
+
+        Ability newSource = source.copy();
+        newSource.setWorksFaceDown(true);
+        MageObjectReference mor = new MageObjectReference(card.getId(),
+                card.getZoneChangeCounter(game) + 1, game);
+        game.addEffect(new BecomesFaceDownCreatureEffect(
+                null, mor, Duration.Custom,
+                BecomesFaceDownCreatureEffect.FaceDownType.MANUAL), newSource);
+        controller.moveCards(card, Zone.BATTLEFIELD, source, game, false, true, true, null);
+
+        // Need to process the move so that any ETB triggers on the card don't fire
+        game.getState().processAction(game);
```

Comment:
No needs here cause moveCards to battlefield already call applyeffect/processaction inside:

![IMG_9623](https://github.com/magefree/mage/assets/8344157/0be70776-454c-486f-b18e-14a1da4b5874)

Reply:
If I leave out that line, bringing [[Ponyback Brigade]] to the battlefield (for example) causes its trigger to fire twice (enters the battlefield or is turned face up). Is that right?

Reply:
Yes, its two different actions and must raise two events (etb as face down + face up):

> return it to the battlefield face down under its owner’s control if it’s a permanent card, then turn it face up.

Reply:
Bug with Pony related to “etb with face down” problem from #11733